quiz/web/src/modules/common/components/Start/Start.tsx

```diff
@@ -0,0 +1,69 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import NavBar from "modules/common/components/Start/NavBar";
+import CachedIcon from "@mui/icons-material/Cached";
+import IconButton from "@mui/material/IconButton";
+import InputAdornment from "@mui/material/InputAdornment";
+
+function Start() {
+    return (
+        <div className="w-screen h-screen">
+            <NavBar />
+            <Container
+                maxWidth="xl"
+                sx={{ display: "flex", flexDirection: "column", alignItems: "center", my: 5, alignContent: "center" }}
+            >
+                <Typography className="m-2" variant="h2" gutterBottom>
+                    Witaj!
+                </Typography>
+                <Typography variant="subtitle1" gutterBottom>
+                    Wpisz swoją nazwę i zacznij grać.
+                </Typography>
+                <Box
+                    component="form"
+                    sx={{
+                        "& .MuiTextField-root": { m: 1, width: "25ch" },
+                    }}
+                    noValidate
+                    autoComplete="off"
+                >
+                    <TextField
+                        id="standard-helperText"
+                        label="losowa nazwa"
+                        type="search"
+                        helperText="Nazwę - zawsze możesz zmienić"
+                        variant="filled"
+                        size="small"
+                        InputProps={{
+                            endAdornment: (
+                                <InputAdornment position="end">
+                                    <IconButton type="button" sx={{ p: "5px", color: "#92A7FD" }} aria-label="search">
```

Comment:
consider updating there for MUI, i deeply hope it's possible :smile:

Reply:
Can you please tell me what exactly are you talking about?
That I made it too complicated and it can be made simpler?